lib/main.dart

```diff
@@ -1,137 +1,37 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-import 'intro_router.dart';
-import 'routes.dart';
-import 'strings.dart';
-import 'style.dart';
-import 'utils/firebase.dart';
-import 'view/acknowledgements_view.dart';
-import 'view/airway/checklist/intubation_checklist_infographic_page.dart';
-import 'view/airway/checklist/intubation_checklist_page.dart';
-import 'view/airway/extubation/extubation_guidance_page.dart';
-import 'view/airway/extubation/extubation_infographic_page.dart';
-import 'view/airway/guidance/intubation_guidance_infographic_page.dart';
-import 'view/airway/guidance/intubation_guidance_page.dart';
-import 'view/disclaimer_view.dart';
-import 'view/home_page.dart';
-import 'view/icu_non_intensivist/als_bls_guide_view.dart';
-import 'view/icu_non_intensivist/daily_round/daily_round_view.dart';
-import 'view/icu_non_intensivist/proning_guide_view.dart';
-import 'view/icu_non_intensivist/tips_junior_staff_view.dart';
-import 'view/icu_non_intensivist/ventilation/ventilation_infographic_page.dart';
-import 'view/icu_non_intensivist/ventilation/ventilation_view.dart';
-import 'view/info_view.dart';
-import 'view/license_view.dart';
-import 'view/ppe/ppe_off_guidance_method_1.dart';
-import 'view/ppe/ppe_off_guidance_method_1_infographic.dart';
-import 'view/ppe/ppe_off_guidance_method_2.dart';
-import 'view/ppe/ppe_off_guidance_method_2_infographic.dart';
-import 'view/ppe/ppe_on_guidance.dart';
-import 'view/ppe/ppe_on_infographic_page.dart';
-import 'view/ppe/ppe_view.dart';
-import 'view/sbs_guide_view.dart';
-import 'view/staff_welfare/your_welfare_view.dart';
+import 'error_reporting.dart';
+import 'wh_app.dart';
 
 void main() {
   // Add this, and it should be the first line in main method to ensure no crashes before runApp()
-  WidgetsFlutterBinding.ensureInitialized();
+  //WidgetsFlutterBinding.ensureInitialized();
```

Comment:
Do we need to comment this out? Does it cause issues? I've found it helps prevent ugly issues at app startup.

Reply:
Oops, I thats a mistake, I meant to put that back.